code/modules/vending/clothesmate.dm

```diff
@@ -43,7 +43,18 @@
 		/obj/item/clothing/neck/tie/blue = 3,
 		/obj/item/clothing/neck/tie/red = 3,
 		/obj/item/clothing/neck/tie/black = 3,
+		/obj/item/clothing/neck/tie/rainbow = 3,
 		/obj/item/clothing/neck/tie/horrible = 3,
+		/obj/item/clothing/neck/tie/trans = 3,
+		/obj/item/clothing/neck/tie/pan = 3,
+		/obj/item/clothing/neck/tie/enby = 3,
+		/obj/item/clothing/neck/tie/bi = 3,
+		/obj/item/clothing/neck/tie/lesbian = 3,
+		/obj/item/clothing/neck/tie/intersex = 3,
+		/obj/item/clothing/neck/tie/gay = 3,
+		/obj/item/clothing/neck/tie/genderfluid = 3,
+		/obj/item/clothing/neck/tie/asexual = 3,
+		/obj/item/clothing/neck/tie/genderfae = 3,
```

Comment:
Hhhhhhhh

Reply:
I mean... had to make them accessible somehow, but could move them to loadouts

Reply:
I would much prefer that, and remind me at some point to make loadouts have the ability to have like a dropdown selection or something

Reply:
Instead of doing this, make it possible to reskin a tie by Alt clicking it